lib/dash/dash_parser.js

```diff
@@ -707,18 +707,26 @@ shaka.dash.DashParser = class {
       // feasible choices for the value.  Nothing older than
       // timeShiftBufferDepth is still available, and anything less than
       // minBufferTime will cause buffering issues.
-      //
-      // We have decided that our default will be the configured value, or
-      // 1.5 * minBufferTime if not configured. This is fairly conservative.
-      // Content providers should provide a suggestedPresentationDelay whenever
-      // possible to optimize the live streaming experience.
-      const defaultPresentationDelay =
-          this.config_.defaultPresentationDelay || minBufferTime * 1.5;
-      const presentationDelay = suggestedPresentationDelay != null ?
-          suggestedPresentationDelay : defaultPresentationDelay;
+      let delay = 0;
+      if (suggestedPresentationDelay != null) {
+        // 1. If a suggestedPresentationDelay is provided by the manifest, that
+        // will be used preferentially.
+        // This is given a minimum bound of segmentAvailabilityDuration.
+        // Content providers should provide a suggestedPresentationDelay
+        // whenever possible to optimize the live streaming experience.
+        delay = Math.min(
+            suggestedPresentationDelay, segmentAvailabilityDuration);
```

Comment:
segmentAvailabilityDuration can be `null` in VOD, so I suggest add something like: `segmentAvailabilityDuration || Infinity`